README.rst

```diff
@@ -1,34 +1,30 @@
-deck plugin for `Tutor <https://docs.tutor.edly.io>`__
+Deck plugin for `Tutor <https://docs.tutor.edly.io>`__
 ######################################################
 
-Awesome administration dashboard and plugin marketplace for Tutor
+This is a plugin for `Tutor`_ that provides an administration dashboard and integrates `Plugin Marketplace`_ in an Open edX platform.
+
+
+.. _Tutor: https://docs.tutor.edly.io
+.. _Plugin Marketplace: https://edly.io/tutor/plugins-and-themes/
 
 
 Installation
 ************
 
-.. code-block:: bash
+Install the plugin using::
 
     pip install git+https://github.com/overhangio/tutor-deck
 
-.. TODO how to package css files?
+.. TODO how to package css files? 
+.. Replace with tutor plugins install tutor-deck after adding to index.
 
-Usage
-*****
-
-.. code-block:: bash
+Then enable it by::
 
     tutor plugins enable deck
 
-Development
-***********
-
-Install locally::
+Install requirements::
```

Comment:
The steps below should only be required in development. Thus, we should keep the "Development" subsection.